macOS/Configuration/App/DuckDuckGoAppStore.xcconfig

```diff
@@ -33,8 +33,15 @@ ENABLE_APP_SANDBOX = YES
 MACOSX_DEPLOYMENT_TARGET = 12.3
 
 PRODUCT_MODULE_NAME = $(APP_STORE_PRODUCT_MODULE_NAME_OVERRIDE:default=$(DEFAULT_PRODUCT_MODULE_NAME))
-PRODUCT_NAME_PREFIX = DuckDuckGo App Store
-PRODUCT_NAME[config=Release][arch=*][sdk=*] = $(RELEASE_PRODUCT_NAME_OVERRIDE:default=$(PRODUCT_NAME_PREFIX))
```

Comment:
Ok this is something I'd like to check with you @ayoy .  Is there any reason we'd want the release builds to ever have the "App Store" suffix?  Otherwise I think we could remove this entirely.